extensions/amp-analytics/0.1/test/test-events.js

```diff
@@ -580,6 +584,7 @@ describes.realWin('Events', {amp: 1}, env => {
               target,
               matchEmptySpec,
               readyPromise,
+              /* createReadyReportPromiseFunc */ null,
```

Comment:
added a test, to check if `createReadyReportPromiseFunc` is a function that return a promise when eventType is `hidden-v3`.